[Konvoy 1.8.5] Release Notes (#4124)
mesosphere/dcos-docs-site: pages/dkp/konvoy/1.8/release-notes/1.8.5/index.md

menuWeight: 50
excerpt: View release-specific information for Konvoy 1.8.5
beta: false
enterprise: false
---
<!-- markdownlint-disable MD034 -->

**D2iQ&reg; Konvoy&reg; version 1.8.5 was released on February 17th, 2022**

[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]

<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. For new customers, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download Konvoy.</p>

## Release summary

This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy.

### Supported versions

| Kubernetes Support | Version |
| ------------------ | ------- |
| **Minimum**  | 1.18.x  |
| **Maximum**  | 1.20.x  |
| **Default**  | 1.20.15  |

### New features and capabilities

#### CentOS 8 EOL

- CentOS 8 has reached end of life (EOL), and thus is no longer supported by Konvoy.

> #### Update Logic

Reviewer: ```suggestion
#### Update logic
```

> - CentOS 8 has been removed from build tooling and testing as it has reached end of life (EOL). CentOS 8 have been replaced with Alma Linux 8 when building RHEL 8.x images.

Reviewer 2: The 'Alma Linux 8' stuff only applies to Konvoy/DKP 2.x.  We're not doing anything about this in Konvoy 1.x, you just can't use CentOS8 anymore.

```suggestion
- CentOS 8 has reached end of life (EOL), and thus is no longer supported by Konvoy.
```

- Changes to the Kubernetes or Image Registry configuration are now applied to control plane machines. (COPS-7088)

> ### Bug Fixes

Reviewer: ```suggestion
### Bug fixes
```

> - Changes to the Kubernetes or Image Registry configuration are now applied to control plane machines.

Reviewer 2: ```suggestion
- Changes to the Kubernetes or Image Registry configuration are now applied to control plane machines. (COPS-7088)
```

#### TFA

- Updated TFA image version to 3.1.0. (COPS-7145)

> ### Known Issues

Reviewer: ```suggestion
### Known issues
```

> - Updated TFA image version to 3.1.0

Reviewer 2: ```suggestion
- Updated TFA image version to 3.1.0. (COPS-7145)
```

-  Prometheus addon upgrades fail due to a missing alertmanager CRD. To work around this issue, manually apply the missing CRD to the cluster prior to upgrading by running: `kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.47.0/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml`

> ### Component versions

Reviewer 3: Are these already updated?

Author: I removed this as it was a fix in a previous set of notes.
